docs/pim/enable_purchasing_products.md

```diff
@@ -45,6 +45,19 @@ ibexa:
                     - poland
 ```
 
+In the `ibexa_storefront.yaml` file, under the `ibexa.system.storefront_group.product_catalog.regions` configuration, regions are set with `default` value. Remember to either exclude this element or extend it by [configuring other regions](enable_purchasing_products.md#configuring-other-regions-and-currencies).
+
+```yaml
+ibexa:
+    system:
+        storefront_group:
+            product_catalog:
+                currencies:
+                    - EUR
+                regions:
+                    - default
```

Comment:
After discussion, the example should be coherent with each others, and `ibexa.system.<siteaccess_or_siteaccess_group>.product_catalog.regions` must use the region defined in `ibexa.repositories`. So, this example should, from my POV, use the regions from the previous and incoming examples. It is also the occasion to show that those regions can be split over several siteaccess and siteaccess groups.
I propose the following example:

```suggestion
ibexa:
    system:
        storefront_group:
            product_catalog:
                currencies:
                    - EUR
                    - PLN
                regions:
                    - germany
                    - poland 
        another_storefront_group:
            product_catalog:
                currencies:
                    - GBP
                regions:
                    - uk
```
With this example, the 3rd example won't crash the storefront.
:warning: Important counter part: This second example can't be used without the third one or it crash the storefront.
There should be an instruction that link this 2nd example with the 3rd.